From: runsisi <runsisi@zte.com.cn>
Date: Mon, 9 May 2016 15:15:31 +0800
Subject: cls_journal: remove duplicated key generation

Signed-off-by: runsisi <runsisi@zte.com.cn>
(cherry picked from commit 715e99c83e1ebc01ceb9eff0f66da8e2b598766a)
(cherry picked from commit 576ff0ca804779d1eb0b1c44d6ce38e0658b6a05)

Resolves: rhbz#1340482
---
 src/cls/journal/cls_journal.cc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/cls/journal/cls_journal.cc b/src/cls/journal/cls_journal.cc
index 6e7d9d5..d5cdfe3 100644
--- a/src/cls/journal/cls_journal.cc
+++ b/src/cls/journal/cls_journal.cc
@@ -566,7 +566,6 @@ int journal_client_register(cls_method_context_t hctx, bufferlist *in,
   }
 
   cls::journal::Client client(id, data);
-  key = key_from_client_id(id);
   r = write_key(hctx, key, client);
   if (r < 0) {
     return r;
